alert/templates/layouts/base.html

```diff
@@ -1,41 +1,30 @@
-{% extends 'bootstrap/base.html' %}
+<!doctype html>
+<html lang="en">
+  <head>
 
+    <link
```

Comment:
Pulling in the stylesheets as described here: https://getbootstrap.com/docs/5.3/getting-started/download/#cdn-via-jsdelivr